sycl/source/detail/scheduler/commands.hpp

```diff
@@ -272,6 +272,12 @@ class Command {
 
   void waitForPreparedHostEvents() const;
 
+  void flushCrossQueueDeps(const std::vector<EventImplPtr> &EventImpls) {
```

Comment:
nit: I would prefer a container-neutral API.
For example, instead of `std::vector<...>` the function can accept a pair of iterators (begin and end).